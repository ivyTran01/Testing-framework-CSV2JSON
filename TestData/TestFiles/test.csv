name, age, some_bool
"ivy", 22, true
tanzil, 25, false
abdullah, null, true