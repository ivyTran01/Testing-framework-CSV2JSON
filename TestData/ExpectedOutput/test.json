[{"name":"ivy"," age":" 22"," some_bool":" true"},{"name":"tanzil"," age":" 25"," some_bool":" false"},{"name":"abdullah"," age":" null"," some_bool":" true"}]